denizen_scripts/survival/repo_link/temporary/world_event.dsc

```diff
@@ -0,0 +1,173 @@
+world_event_config:
+  type: data
+  materials:
+    iron_ingot: 1
+    iron_block: 9
+  server_amount_needed: 250000
+  personal_amount_needed: 15000
+
+currency_display_item:
+  type: item
+  debug: false
+  material: name_tag
+  display name: <&a>
+  mechanisms:
+    custom_model_data: 3
+
+world_event_bar_0:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>0%
+  mechanisms:
+    custom_model_data: 3
+
+world_event_bar_1:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>20%
+  mechanisms:
+    custom_model_data: 4
+
+world_event_bar_2:
+  type: item
+  debug: false
+  material: beetroot
+  display name: <&a>40%
```

Comment:
percent signs can throw ancient-deprecated errors
```suggestion
  display name: <&a>40<&pc>
```

Reply:
It take two percentage symbols to trigger said error